<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridCreatingConditionTriangleSettingDialog</class>
 <widget class="QDialog" name="GridCreatingConditionTriangleSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid Creation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="angleCheckBox">
         <property name="text">
          <string>Specify minimum angle for cell vertices</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="TooltipLabel" name="angleHelpLabel">
         <property name="toolTip">
          <string>By specifying minimum angle, you can avoid generating thin and long triangle cells.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="angleSpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>32.000000000000000</double>
         </property>
         <property name="value">
          <double>20.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="angleLabel">
         <property name="text">
          <string>degrees</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="angleWarningLabel">
           <property name="text">
            <string>When polygon has vertices that has smaller angle than the value above, the condition is not applyed to the cells that include those vertices.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="areaCheckBox">
         <property name="text">
          <string>Specify maximum area for cells</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="TooltipLabel" name="areaHelpLabel">
         <property name="toolTip">
          <string>By specifying small values, you can generate fine grids.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../guibase/guibase.qrc">:/libs/guibase/images/iconQuestion.svg</pixmap>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="RealNumberEditWidget" name="areaEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="areaLabel">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS UI Gothic'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;m&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>45</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TooltipLabel</class>
   <extends>QLabel</extends>
   <header>guibase/tooltiplabel.h</header>
  </customwidget>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../guibase/guibase.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GridCreatingConditionTriangleSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GridCreatingConditionTriangleSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>areaCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>areaEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angleSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
